graviti/client/request.py

```diff
@@ -7,15 +7,19 @@
 
 import logging
 from typing import Any
+from uuid import uuid4
 
 from requests.models import Response
+from tensorbay.exception import ResponseError, ResponseErrorDistributor
+from tensorbay.utility import config, get_session
+
+from graviti.__version__ import __version__
 
 logger = logging.getLogger(__name__)
+PARTIAL_URL = "gatewayv2/tensorbay-open-api/v1"
 
 
-def open_api_do(
-    url: str, access_key: str, method: str, **kwargs: Any  # pylint: disable=unused-argument
-) -> Response:
+def open_api_do(url: str, access_key: str, method: str, **kwargs: Any) -> Response:
     """Send a request to the TensorBay Open API.
```

Comment:
`OpenAPI` without space between `Open` and `API`
Do not use `tensorbay` anymore